File: A03/main.py
N, K = map(int, input().split())
P = tuple(map(int, input().split()))
Q = tuple(map(int, input().split()))

ans = "No"
for x in P:
    for y in Q:
        if x + y == K:
            ans = "Yes"
            break
    else:
        continue

print(ans)

File: A03/main.rs
use proconio::input;

fn main() {
    input! {
        N: i64,
        K: i64,
        P: [i64; N],
        Q: [i64; N],
    }

    let mut ans = "No";
    'p_loop: for pn in P {
        'q_loop :for qn in &Q {
            if (pn + qn) == K {
                ans = "Yes";
                break 'p_loop;
            }
        }
    }

    println!("{}", ans);
}